lib/psci/psci_suspend.c

```diff
@@ -292,7 +292,7 @@ void psci_cpu_suspend_finish(unsigned int cpu_idx,
 	 * Dispatcher to let it do any bookeeping. If the handler encounters an
 	 * error, it's expected to assert within
 	 */
-	if (psci_spd_pm && psci_spd_pm->svc_suspend) {
+	if (psci_spd_pm && psci_spd_pm->svc_suspend_finish) {
```

Comment:
Good spot! I guess we've been getting away this since usually SPDs with a registered svc_suspend_finish() handler also have a registered svc_suspend() handler.